camel/agents/chat_agent.py

```diff
@@ -106,8 +105,7 @@ class ChatAgent(BaseAgent):
     def __init__(
         self,
         system_message: BaseMessage,
-        model_type: Optional[ModelType] = None,
-        model_config: Optional[BaseConfig] = None,
+        model: Optional[BaseModelBackend] = None,
```

Comment:
What does `Backend` suffix mean in this type?

Reply:
pls check `BaseModelBackend` under "camel/models" folder for the backend functionality we implemented